### PR TITLE
add prequisities for anagram practice exercise

### PR DESCRIPTION
**Pull request by AnishDe12020**
I have added the prerequisites for the anagram practice exercise. I am opening this pull request only for this exercise and as per how it goes I will decide on how many exercises I should add per pull request from next time. 

References #984

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -764,7 +764,7 @@
         "name": "Anagram",
         "uuid": "309fa4f1-e03a-4ab2-b371-cdf742501cf7",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["strings", "arrays", "basics", "for-loops", "conditionals", "comparison"],
```
Comment (SleeplessByte) [inline config.json:767]: Because `basics` is the prerequisite of all "other things", it doesn't need to be listed here. 

Otherwise, great! Thanks :D

Comment (AnishDe12020) [reply config.json]: Sure, removed it in https://github.com/exercism/javascript/pull/1548/commits/739d8f23adcb4eb75b39c6d3da18b308f69adda6

**Comment by junedev** [comment]
@AnishDe12020 The CI checks complain about the formatting. Here you can find out how to format the content correctly: https://github.com/exercism/javascript/blob/main/CONTRIBUTING.md#format

**Comment by AnishDe12020** [comment]
> @AnishDe12020 The CI checks complain about the formatting. Here you can find out how to format the content correctly: https://github.com/exercism/javascript/blob/main/CONTRIBUTING.md#format

Ya sure, I have run the script to format. Just one recommendation, running prettier can be added as a script to `package.json` so that it can easily be run with a comment like `npm run format`.

**Pull request merged**